website/app/components/doc/page/header/index.hbs

```diff
@@ -21,24 +22,28 @@
           discover the search results.</div>
         <button
           type="button"
-          class="doc-page-header__search-button"
+          class="doc-page-header__desktop-icon-item"
           data-doc-algolia-search-autocomplete-secondary-trigger
           aria-describedby="search-button"
+          aria-label="Search"
         >
-          <Hds::Icon @name="search" />
-          Search
+          <Hds::Icon @name="search" @size="24" />
```

Comment:
Why did we want to replace words in the navigation controls with just icons? To me, having just icons makes things less clear and also there seemed to be plenty of room for including the words.

Reply:
Specifically [for avoiding this scenario](https://www.figma.com/design/ZlIlup8HxCOlQLXRwv56In/HDS-Content-Guidelines?node-id=3-12306&t=uKAjr2LKxywzroAz-1), so instead we get [this](https://www.figma.com/design/ZlIlup8HxCOlQLXRwv56In/HDS-Content-Guidelines?node-id=119-3749&t=uKAjr2LKxywzroAz-1)

Reply:
Could use a breakpoint and just hide the words at a certain point. Just saying...